templates/kafka-pv.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2020 Crown Copyright
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: {{ .Values.global.persistence.kafka.name }}
+  {{- if eq .Values.hosting "aws" }}
+  labels:
+    type: amazonEBS
+  {{- end }}
+spec:
+  accessModes:
+  - ReadWriteMany
+  - ReadWriteOnce
```

Comment:
ReadWriteMany offers: the volume can be mounted as read-write by many nodes
whereas ReadWriteOnce offers the volume can be mounted as read-write by a single node
Therefore, - ReadWriteOnce should be deleted